src/main/java/github/weichware10/toolbox/Main.java

```diff
@@ -50,20 +70,182 @@ public void handle(WindowEvent event) {
+        boolean available = dataBaseClient.trials.getAvailability(trialId);
+        if (!available) {
+            warnText.setVisible(true);
+            return;
+        }
+
+        // Config laden
+        boolean success = configClient.loadFromDataBase(trialId);
+        if (!success) {
+            warnText.setVisible(true);
+            return;
+        }
+
+        // Tutorial noch hinzufügen an dieser Stelle
+        TestVorbildschirm.display(primaryStage, configClient);
+    }
+
+    protected void changeDb() {
+        Dialog<Void> dialog = new Dialog<>();
+        ButtonType okButtonType = new ButtonType("OK", ButtonData.OK_DONE);
+        dialog.getDialogPane().getButtonTypes().addAll(okButtonType, ButtonType.CANCEL);
+        FXMLLoader loader = new FXMLLoader(getClass().getResource("DataBaseDialog.fxml"));
+
+        Parent root = null;
+        try {
+            root = loader.load();
+        } catch (IOException e) {
+            Logger.error("Error while displaying dialog", e);
+        }
+        if (root == null) {
+            return;
+        }
+        DataBaseDialogController controller = loader.getController();
+        // nur bei Erfolg Dialog verlassen
+        final Button ok = (Button) dialog.getDialogPane().lookupButton(okButtonType);
+        ok.addEventFilter(ActionEvent.ACTION, okEvent -> {
+            try {
+                // Versuchen Datenbankclient zu erstellen
+                dataBaseClient = new DataBaseClient(
+                        controller.getUrl(),
+                        controller.getUsername(),
+                        controller.getPassword(),
+                        controller.getSchema());
+            } catch (IllegalArgumentException e) {
+                // display error
+                Logger.info("Error while changing database connection");
+                controller.setWarning("Your input is not valid:");
+                controller.setError(e.getMessage());
+                okEvent.consume();
+            }
+            // erstellt den Config Client um die Informationen aus der Config zu handeln
+            configClient = new ConfigClient(dataBaseClient);
+        });
+        dialog.getDialogPane().setContent(root);
+        dialog.showAndWait();
+    }
+
+    protected void createZoomMapsTestTrial(TextField trialIdInput) {
+        if (dataBaseClient == null) {
+            return;
+        }
+        ZoomMapsConfiguration zoomMapsConfiguration = new ZoomMapsConfiguration(
+                4.2f,
+                true,
+                Arrays.asList(
+                        "https://media.discordapp.net/attachments/707505654218358818/836645130417078282/TBY5IslL.png",
+                        "url2", "url3"));
+
+        createTestTrial(new Configuration(
+                "dunno yet",
+                "Test Question?",
+                zoomMapsConfiguration),
+                trialIdInput);
+    }
+    protected void createCodeChartsTestTrial(TextField trialIdInput) {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.EmptyLineSeparatorCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'METHOD_DEF' should be separated from previous line.